### ISSUE TITLE
Typscript support

### ISSUE THREAD

**Issue by MiguelMadero** [burkeholland/simple-react-snippets#8]
I just installed this and realized it doesn't work with TS and I was going to send a PR to make all the current ones work in TS. I don't think most of them really need types, like ss or impt, those should work the same for js(x)|ts(x) others may have slight variations. 

What do you think? Should I send another PR? 

BTW, I saw #4 , but the scope looks a bit different.

**Comment by burkeholland** [comment]
Yes! Send it in. I've been hesitant to add .ts support, but more and more people are requesting it.

**Comment by burkeholland** [comment]
@blurbyte is making all your TypeScript dreams come true!

Fixed in #19

**Comment by mfour-merickson** [comment]
Hey @burkeholland Thanks for the push, it is now "partially" working :-)

When using with `*.tsx` files, all works as expected.
However, if using with `*.ts` files, it does not work as expected

@MiguelMadero Did you apply it to both `*.ts` and `*.tsx`?

**Comment by burkeholland** [comment]
Thanks @mfour-merickson! I can add a mapping for TypeScript. Right now we look like this...

```
"snippets": [{
        "language": "javascriptreact",
        "path": "./snippets/snippets.json"
      },
      {
        "language": "javascript",
        "path": "./snippets/snippets.json"
      },
      {
        "language": "typescriptreact",
        "path": "./snippets/snippets-ts.json"
      }
    ]
```

**Comment by burkeholland** [comment]
Fixed in 1.2.2. Holding this issue for confirmation from @mfour-merickson.

**Comment by mfour-merickson** [comment]
@burkeholland Sorry for delay on confirmation, all is good now :-)
